django/core/management/utils.py

```diff
@@ -169,7 +170,11 @@ def run_formatters(written_files, black_path=(sentinel := object())):
     if black_path is sentinel:
         black_path = shutil.which("black")
     if black_path:
-        subprocess.run(
-            [black_path, "--fast", "--", *written_files],
-            capture_output=True,
-        )
+        try:
+            subprocess.run(
+                [black_path, "--fast", "--", *written_files],
+                capture_output=True,
+            )
+        except OSError:
+            logger = logging.getLogger(__name__)
```

Comment:
Currently Django has a complete lists of the loggers it provides, these are documented in: https://docs.djangoproject.com/en/5.1/ref/logging/#django-logging-extensions

While using `__name__` is a standard practice, I don't think this is consistent with other usages of logging within Django core. I think we need to create and document new logger for the Django-provided management commands. Perhaps "django.management"?

Reply:
Ah right, we can do better than `__name__`.

Looking again, I don't know why it didn't occur to me to reuse `self.stderr`. If people are carefully piping stdout and stderr to certain places (which I recall from ticket-29026), I bet they'll want to keep everything siloed into those streams.

What do you think about the approach in 26e85ae?

Reply:
(IMO that also has the advantage of avoiding the need to create a new logger that might sound like it's at cross purposes with the ordinary self.stdout/self.stderr pattern for management commands.)